jumpapp

```diff
@@ -187,6 +191,41 @@ get_prev_window() {
     fi
 }
 
+get_last_focused_window() {
+    local matched_windows=($@)
+    local recent_windows=($(xprop -root | grep "^_NET_CLIENT_LIST_STACKING" | cut -d '#' -f2- | tr ',' ' '))
+    local active_window=$(get_active_windowid)
+    local recent_window
+
+    for ((i=${#recent_windows[@]}-1; i>=0; i--)) do
+
+        recent_window=${recent_windows[i]}
+        if are_numbers_equal $recent_window $active_window; then
+            continue
+        fi
+
+        for matched_window in ${matched_windows[@]}; do
+            if are_numbers_equal $recent_window $matched_window; then
+                printf '%s\n' $matched_window
+                return
+            fi
+        done
+    done
+
+    get_next_window "$@"
```

Comment:
Nice. It seems like a good idea to have a fallback in case the window manager doesn't support `_NET_CLIENT_LIST_STACKING` correctly.